start_esp/start_esp.py

```diff
@@ -613,6 +616,15 @@ def make_argparser():
     body is larger than the buffer, the whole body or only its part is
     written to a temporary file.''')
 
+    parser.add_argument('--large_client_header_buffers', default=DEFAULT_LARGE_CLIENT_HEADER_BUFFERS, help='''
+    Sets the maximum number and size of buffers used for reading large client
+    request header. A request line cannot exceed the size of one buffer, or the
+    414 (Request-URI Too Large) error is returned to the client. A request header
+    field cannot exceed the size of one buffer as well, or the 400 (Bad Request)
+    error is returned to the client.
+    http://nginx.org/en/docs/http/ngx_http_core_module.html#large_client_header_buffers
```

Comment:
Please also mention the format of value.  It has a space in between,  I am wondering if space will cause problem for flag passing.  maybe we should not use space.

Reply:
Yeah one needs to quote it. I will mention that in the flag. 

I think it's still good to keep in the same format as nginx value introducing our own formatting can be more confusing.